config/locales/en.yml

```
@@ -117,3 +117,15 @@ en:
           logo_present: 'Logo present'
           logo_absent: 'Logo absent'
   log_in_first: 'Please log in first'
+  companies:
+    labels:
+      years_text: 'years on the market'
```

maybe should also implement I18n pluralisation 
```suggestion
      years_text:
        one:  'year on the market'
        other:  'years on the market'
```

---

Thanks for the great suggestion!